Makefile

```diff
@@ -12,14 +12,16 @@ LIBPCAP_ARCH ?= x86_64-unknown-linux-gnu
 # For compiling libpcap and CGO
 CC ?= gcc
 ARCHS ?= amd64 arm64
+CGO_CFLAGS="-I/pwru/libpcap"
+CGO_LDFLAGS="-L/pwru/libpcap -lpcap -static"
```

Comment:
Failed to compile locally, the path should be `./libpcap`.

Reply:
What command did you run locally? `make release` and `make` are both okay on my local ubuntu.

Reply:
I don't know why this is supposed to work locally. The CI build with docker which set workdir to /pwru so there is no problem.

Reply:
I agree that this shouldn't be an absolute path, otherwise e.g. `make pwru` will fail on hosts where `/pwru/libpcap/` does not exist. We could also consider using `-L$(CURDIR)/libpcap`.